### PR TITLE
fix #151 adding Basics.logBase and Basics.e to morphir-sdk

### LINKED ISSUE #151: Basics.logBase and Basics.e are missing from the SDK
**Describe the bug**
Basics.logBase and Basics.e are missing from the SDK

**Expected behavior**
These functions should be implemented in the SDK and they should behave the same way as the functions with the same name in elm/core.

### PR DESCRIPTION
**Pull request by Abdul-Hameed-Turntabl**

### CODE REVIEW

File: morphir/sdk/core/src/morphir/sdk/Basics.scala [hunk 1]
```diff
@@ -108,6 +108,10 @@ object Basics {
     if (a < min) min
     else if (a > max) max
     else a
+  @inline def logBase(base: Float, number: Float): Float =
+    divide(scala.math.log(number))(scala.math.log(base))
+  @inline def e(a: Float): Float =
+    scala.math.exp(a)
```
Comment (AttilaMihaly) [inline morphir/sdk/core/src/morphir/sdk/Basics.scala:114]: This doesn't look right. `e` just returns a number in Elm: https://package.elm-lang.org/packages/elm/core/latest/Basics#e

Comment (AttilaMihaly) [reply morphir/sdk/core/src/morphir/sdk/Basics.scala:114]: `Math.E` looks like the right thing to use here: https://docs.oracle.com/javase/8/docs/api/java/lang/Math.html#E

**Review by AttilaMihaly — CHANGES_REQUESTED** [review]

File: morphir/sdk/core/src/morphir/sdk/Basics.scala (hunk 1 above)
Comment (Abdul-Hameed-Turntabl) [reply morphir/sdk/core/src/morphir/sdk/Basics.scala:114]: Okay @AttilaMihaly, I will make the necessary changes.

Comment (Abdul-Hameed-Turntabl) [reply morphir/sdk/core/src/morphir/sdk/Basics.scala]: @AttilaMihaly Just I mistook e() for exp() math function

**Review by stephengoldbaum — APPROVED** [review]

**Review by Abdul-Hameed-Turntabl — COMMENTED** [review]
Basics.e has successfully been added

**Review by AttilaMihaly — APPROVED** [review]

**Comment by Abdul-Hameed-Turntabl** [comment]
@AttilaMihaly, the checks seen to fail at Basics.power() for float type. I intend to change it.

**Pull request merged**